fm2prof/RegionPolygonFile.py

```diff
@@ -91,7 +93,7 @@ def classify_points_with_property_shapely_prep(
         return np.array(points_regions)
 
     def classify_points_with_property_rtree_by_polygons(
-        self, iterable_points: Iterable[list], property_name: str = "name"
+        self, points: Iterable[list], property_name: str = "name"

```

Comment:
Shouldn't this be an `Iterable[Point]` instead of `Iterable[list]`?

Reply:
Yes, missed that.